Include/internal/pycore_signal.h

```diff
@@ -38,12 +37,12 @@ PyAPI_FUNC(void) _Py_RestoreSignals(void);
 #define INVALID_FD (-1)
 
 struct _signals_runtime_state {
-    volatile struct {
-        _Py_atomic_int tripped;
+    struct {
+        int tripped;
         /* func is atomic to ensure that PyErr_SetInterrupt is async-signal-safe
          * (even though it would probably be otherwise, anyway).
          */
```

Comment:
I suggest to remove the comment here, and *maybe* add a comment in set_handler() to mention the relationship with PyErr_SetInterrupt().

Reply:
Done